v2/pkg/engine/resolve/loader.go

```diff
@@ -971,6 +973,30 @@ func (l *Loader) renderErrorsFailedToFetch(fetchItem *FetchItem, res *result, re
 	if err != nil {
 		return err
 	}
+
+	fmt.Printf("")
+
+	if l.apolloRouterCompatibilitySubrequestHTTPErrror && (res.statusCode < 200 || res.statusCode > 300) {
```

Comment:
Does apollo really do this for 200–300?

Reply:
that's not what this says, it's maybe just a bad way of writing this, its anything other than 200-300

Reply:
but now that you mention it, it should probably just be >=400, i never black boxed sub-200 response codes against router or the 3XX range

Reply:
sorry i misread
but it's probably just > 399

Reply:
fixed